media-gfx/yafaray/files/yafaray-3.3.0-blender-exporter-paths.patch

```diff
@@ -0,0 +1,15 @@
+diff --git a/work/Blender-Exporter-3.3.0/__init__.py b/work/Blender-Exporter-3.3.0/__init__.py
+index 85dc1c2..70a79e8 100644
+--- a/__init__.py
++++ b/__init__.py
+@@ -22,8 +22,8 @@ import sys
+ import os
+ import ctypes
+ 
+-PLUGIN_PATH = os.path.join(__path__[0], 'bin', 'yafaray-plugins')
+-BIN_PATH = os.path.join(__path__[0], 'bin')
++PLUGIN_PATH = '/usr/LIBDIR/yafaray-plugins'
```

Comment:
How about making it `/usr/lib/yafaray-plugins` to avoid sed-ing it?

Reply:
I was thinking multilib but that kind of makes no sense here.